pkg/cmd/helm/release/release_test.go

```diff
@@ -230,7 +231,7 @@ func TestStepHelmReleaseWithOCIUsingUserName(t *testing.T) {
 			CLI: "helm registry login " + OCIRegistry + " --username " + o.RepositoryUsername + " --password " + o.RepositoryPassword,
 		},
 		fakerunner.FakeResult{
-			CLI: "helm push myapp-" + chartVersion + ".tgz " + OCIRegistry,
+			CLI: "helm push myapp-" + chartVersion + ".tgz " + OCIRegistry + " --registry-config /tekton/creds-secrets/tekton-container-registry-auth/.dockerconfigjson",
```

Comment:
Same here? o.RegistryConfigFile

Reply:
ok

Reply:
quite smooth to get the comments in the file immediately. VS Code